### PR TITLE
PG: fix "name" field length (63 instead of 64)

### PR DESCRIPTION
**Pull request by elpaso**

### CODE REVIEW

File: src/providers/postgres/qgspostgresprovider.cpp [hunk 1]
```diff
@@ -1123,8 +1123,8 @@ bool QgsPostgresProvider::loadFields()
       else if ( fieldTypeName == QLatin1String( "name" ) )
       {
         fieldSubType = QVariant::String;
-        // "name" type lenght is 64 according to: SELECT typlen FROM pg_type WHERE typname = 'name';
-        fieldSize = 64;
+        // "name" type lenght is 63
```
Comment (stev-0) [inline src/providers/postgres/qgspostgresprovider.cpp:1126]: typo - switch to **length**

Comment (stev-0) [inline src/providers/postgres/qgspostgresprovider.cpp:1126]: typo - switch to length

Comment (pblottiere) [reply src/providers/postgres/qgspostgresprovider.cpp:1126]:
```suggestion
        // "name" type length is 63
```

Comment (pblottiere) [reply src/providers/postgres/qgspostgresprovider.cpp:1126]: @stev-0 FYI you can use GitHub "Suggestions" for this kind of comment ^.

And thanks for raising this :+1:

Comment (elpaso) [inline src/providers/postgres/qgspostgresprovider.cpp:1126]:
```suggestion
```

**Pull request merged**